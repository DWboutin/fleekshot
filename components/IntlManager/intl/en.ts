import flat from "flat";

import signUpForm from "../../SignUpForm/intl/en";
import signInForm from "../../SignInForm/intl/en";
import { IntlMessages } from "./types";

const en = {
  hello: "Hello",
  ...signUpForm,
  ...signInForm,
};

export default flat(en, { delimiter: "_" }) as IntlMessages;
