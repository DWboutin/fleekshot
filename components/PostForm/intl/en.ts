import { PostFormIntl } from "./type";

const en: PostFormIntl = {
  postForm: {
    uploadImage: "Upload image",
    share: "Share",
    inputLabel: "Write your new post",
    errors: {
      tooShort: "Is too short",
      tooLong: "Is too long",
      isRequired: "Is required",
      imageIsRequired: "An image is required",
    },
  },
};

export default en;
