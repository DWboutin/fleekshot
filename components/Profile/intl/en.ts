import { ProfileIntl } from "./type";

const en: ProfileIntl = {
  profile: {
    title: "Profile",
    uploadPicture: "Upload picture",
  },
};

export default en;
