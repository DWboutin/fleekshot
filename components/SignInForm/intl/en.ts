import { SignInFormIntl } from "./type";

const en: SignInFormIntl = {
  signInForm: {
    title: "Sign in",
    input: {
      username: {
        label: "Username",
      },
      password: {
        label: "Password",
      },
    },
    button: {
      submit: "Sign in",
    },
    message: {
      success: "You're now signed in and will be redirected",
    },
    errors: {
      tooShort: "Is too short",
      tooLong: "Is too long",
      isRequired: "Is required",
      notStrongEnough: "Is not strong enough",
      mismatch: "Password mismatch",
      noUser: "No user was found",
    },
  },
};

export default en;
