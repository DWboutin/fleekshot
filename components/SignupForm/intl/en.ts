import { SignupFormIntl } from "./type";

const en: SignupFormIntl = {
  signupForm: {
    input: {
      name: {
        label: "Complete name",
      },
      username: {
        label: "Username",
      },
      password: {
        label: "Password",
      },
      confirmPassword: {
        label: "Confirm password",
      },
    },
    errors: {
      tooShort: "Too short",
      tooLong: "Too long",
      isRequired: "Is required",
      notStrongEnough: "Not strong enough",
      mismatch: "Password mismatch",
    },
  },
};

export default en;
