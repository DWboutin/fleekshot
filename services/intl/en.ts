import { ErrorCodes } from "../../server/handler/errorHandler";
import { HttpErrorsIntl } from "./type";

const en: HttpErrorsIntl = {
  httpErrors: {
    [ErrorCodes.DuplicateInDatabase]:
      'Duplicate "{field}" field with "{value}" in database',
    [ErrorCodes.ValidationError]: 'Field "{field}" {message}',
    [ErrorCodes.NoUser]: "Sign in error - {message}",
    field: {
      name: "name",
      username: "username",
      password: "password",
      confirmPassword: "confirm password",
    },
  },
};

export default en;
