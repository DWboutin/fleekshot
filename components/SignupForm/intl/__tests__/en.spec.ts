import en from "../en";

describe("intl SignupForm en", () => {
  it("should match snapshot", () => {
    expect(en).toMatchSnapshot();
  });
});
