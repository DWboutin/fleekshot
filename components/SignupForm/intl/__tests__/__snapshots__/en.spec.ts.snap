// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`intl SignUpForm en should match snapshot 1`] = `
Object {
  "signUpForm": Object {
    "button": Object {
      "submit": "submit",
    },
    "errors": Object {
      "isRequired": "Is required",
      "mismatch": "Password mismatch",
      "notStrongEnough": "Not strong enough",
      "tooLong": "Too long",
      "tooShort": "Too short",
    },
    "input": Object {
      "confirmPassword": Object {
        "label": "Confirm password",
      },
      "name": Object {
        "label": "Complete name",
      },
      "password": Object {
        "label": "Password",
      },
      "username": Object {
        "label": "Username",
      },
    },
    "title": "Sign up",
  },
}
`;
